support/Environments/orca_gcc.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_setup_modules() {
+    echo "All modules on orca are provided by the system"
+}
+
+spectre_unload_modules() {
+module unload python/2.7.8
+module unload spack
+module unload gcc/7.3.0
+module unload openmpi/3.1.0
+module unload blaze-3.2-gcc-7.3.0-u6l3rrb
+module unload boost-1.67.0-gcc-7.3.0-e6bshzw
+module unload brigand-master-gcc-7.3.0-zizme5m
+module unload catch-2.2.1-gcc-7.3.0-fhnfleh
+module unload gsl-2.4-gcc-7.3.0-okfp45i
+module unload hdf5-1.10.2-gcc-7.3.0-7erzlsy
+module unload jemalloc-4.5.0-gcc-7.3.0-crqk7yv
+module unload openblas-0.2.20-gcc-7.3.0-rjpf3cb
+module unload yaml-cpp-develop-gcc-7.3.0-wy6dpng
+module unload libxsmm/1.9
+module unload cmake-3.11.3-gcc-7.3.0-isxbgez
+module unload charm
+}
+
+spectre_load_modules() {
+module load python/2.7.8
```

Comment:
We don't support python 2, you should update it. It's EOL is coming up on Jan. 1 2020, so if you plan on having Orca around longer than that you should update.

Reply:
I used python2 because spack calls “python”, not “python3.” I couldn’t figure out how to use python3 with spack, but we do have python3 on orca and can switch when necessary. Suggestions on how to get spack going with python3 are welcome.

Reply:
I'm quite surprised by this because I use Spack v0.11.2 with python3.6.5 on my laptop without troubles. Older versions of spack, about 1 year ago, did not work with python3 but that seems to be resolved. It might be that you need 3.5 or 3.6. That's about all I can tell you. Do you have python 2.7.12 or newer? That would be a reasonable alternative for the time being.

Reply:
Thanks very much, @nilsdeppe. When I installed python 3, the executable was named python3, but spack kept running “python”, which was python 2. Maybe it would work with some configuring? I don’t have 2.7.12 but can easily get it via spack.

Reply:
You should be able to just change the python used in `bin/spack`. That's what I've done in the past before they supported python3. My `python` was 3, so I changed `bin/spack` to use `python2` instead of `python`. The opposite should work too :)